### ISSUE TITLE
[Vue warn]: Failed to mount component: template or render function not defined.

### ISSUE THREAD

**Issue by pengteling** [SevenOutman/vue-aplayer#19]
```
<template>
  <div class="hello">
    <h1>{{ msg }}</h1>
    <aplayer autoplay :music="{
      title: 'Preparation',
      author: 'Hans Zimmer/Richard Harvey',
      url: 'http://devtest.qiniudn.com/Preparation.mp3',
      pic: 'http://devtest.qiniudn.com/Preparation.jpg',
      lrc: '[00:00.00]lrc here\n[00:01.00]aplayer'
    }"></aplayer>
  </div>
</template>

<script>
import Aplayer from 'vue-aplayer'
export default {
  name: 'HelloWorld',
  components: {
    Aplayer
  },
  data () {
    return {
      msg: 'Welcome to Your Vue.js App'
    }
  }
}
</script>
```

[Vue warn]: Failed to mount component: template or render function not defined.
found in

---> <Aplayer>
       <Anonymous>
         <App> at src\App.vue
           <Root>

**Comment by kangxiaojun** [comment]
Failed to mount component: template or render function not defined

**Comment by nperegrine** [comment]
I am getting the same issue, did anyone figure out a solution to this?

**Comment by upnow** [comment]
The same issue as yours. I've tried several ways but it showed nothing.

**Comment by Mr-Six-Hu** [comment]
I have  got the same issue .

**Comment by SevenOutman** [comment]
@pengteling There might be something wrong with my production build config, I'm investing right now. A quick fix to this is to `import Aplayer from 'vue-aplayer/src/vue-aplayer.vue'`, which requires you have `sass-loader` installed and.

**Comment by upnow** [comment]
I've handled this issue. You should copy the files in his src folder to your own project. And be careful for the path of those files you quoted in your *.vue.

**Comment by Mr-Six-Hu** [comment]
*{
        margin: 0;
        padding: 0;
    }
    font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    


thanks，but i have another issue，the progress bar did not work ，do you have the same problem ?

    
        a#ntes-pcmail-signature-default:hover {
            text-decoration: underline;
            color: #199cff;
            cursor: pointer;
        }
        a#ntes-pcmail-signature-default:active {
            text-decoration: underline;
            color: #246fce;
            cursor: pointer;
        }
    



    On 2/1/2018 15:30，Jeff<notifications@github.com> wrote： 


I've handled this issue. You should copy the files in his src folder to your own project. And be careful for the path of those files you quoted in your *.vue.

**Comment by Mr-Six-Hu** [comment]
thanks，but i have another issue，the progress bar did not work ，do you have the same problem ?

**Comment by upnow** [comment]
It worked well in my project. It seems that aplayer-controller-progress.vue is responsible for the job. Have a look.

**Comment by SevenOutman** [comment]
@pengteling @kangxiaojun @nperegrine @upnow @Mr-Six-Hu 
This issue was caused by a missing `output.libraryTarget` in my webpack build config. It's been fixed in npm version 1.0.4.
